bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/PwaManifestServlet.java

```diff
@@ -148,11 +147,11 @@ private JsonArray getIcons(final Configuration configuration) {
 
                         json.addProperty(KEY_ICON_SRC, path);
                         json.addProperty(KEY_ICON_SIZE, p.get(PN_ICON_SIZE, String.class));
-                        json.addProperty(KEY_ICON_TYPE, mimeTypeService.getMimeType(path));
+                        json.addProperty(KEY_ICON_TYPE, "png");
```

Comment:
Doesn’t have to be png - ideally these are svg and serve all sizes.

Reply:
Agreed but I see an error if it's not png "Site cannot be installed: a 144px square PNG icon is required, but no supplied icon meets this requirement" . Let me know if that's the case for you

Reply:
https://developer.mozilla.org/en-US/docs/Web/Manifest#icons